tests/testthat/test-fit_model.R

```diff
@@ -0,0 +1,32 @@
+library(testthat)
+library(tidymodels)
+library(dplyr)
+source("../../R/fit_model.R")
+
+testthat_data <- tibble(
+  x = 1:10,
+  z = 2:11,
+  y = 2 * (1:10) + 1 * (2:11)
+)
+
```

Comment:
I would suggest moving your examples in a helper-fit_model.R function because I believe that's the format that we have to follow. So, all of your examples should probably be moved to another helper function and it can stay in this same folder tests/testthat. You can look at Edwin's helper function as a guide maybe.

Reply:
After asking the professor, we really only need helper functions if our test examples are really long and extensive. Given how short your examples are Marcela, I think that you don't really need a helper function.